ConorMFinn/CO600-SeriousCyberSec
# Fix General Bugs

**Description of work.**
Fixed some minor bugs, check commit messages. 

**To test:**
Check object collisions and sorting 
CHeck symbol appears on all interactable objects. 

Fixes #89 
Fixes #88
